ci-operator/step-registry/ipi/conf/featureset/ipi-conf-featureset-commands.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
We do not need this step to set `TechPreviewNoUpgrade`, but set `FEATURE_SET: TechPreviewNoUpgrade` env in job config level.

Reply:
@jianlinliu Thanks, didn't notice this variable.